02-set-up.Rmd

```diff
@@ -177,8 +177,15 @@ identical(r1, r2)
 
 ## R version
 
-It is important to keep your R installation and packages up-to-date.
-This section explains how.
+It is important to be aware that R is an evolving software project, whose behaviour changes over time.
+This applies to an even greater extent to packages, the behaviour of which can change substantially from one release to the next.
+Depending on context, you may want to your R installation and packages up-to-date, or 'pin' the version of R and packages to specific versions, to ensure stability.
+For most desktop users, keeping packages up-to-date is desirable: new code tends to be more efficient, intuitive, robust and feature rich. This section explains how.
+
+```{block, packrat, style='rmdtip'}
+Previous R versions can be installed from CRAN's archive or previous R releases. The binary versions for all OSs can be found at [cran.r-project.org/bin/](https://cran.r-project.org/bin/). To download binary versions for Ubuntu 'Wily', for example, see [cran.r-project.org/bin/linux/ubuntu/wily/](https://cran.r-project.org/bin/linux/ubuntu/wily/). To 'pin' specific versions of R packages you can use the **packrat** package. For more on pinning R versions and R packages see articles on RStudio's website [Using-Different-Versions-of-R](https://support.rstudio.com/hc/en-us/articles/200486138-Using-Different-Versions-of-R) and [rstudio.github.io/packrat/](https://rstudio.github.io/packrat/).
```

Comment:
It's a technical term (e.g. see [here](https://help.ubuntu.com/community/PinningHowto)), but not one that's [used in packrat](https://github.com/rstudio/packrat/search?utf8=%E2%9C%93&q=pin).